### PR TITLE
Add license

### LINKED ISSUE #1210: License
There doesn't seem to be a license specified, wouldn't that be nice to have? Someone on Gitter asked if the ML tools were open source.

### PR DESCRIPTION
**Pull request by simonbray**
Fix #1210.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2012-present Björn Grüning and other contributors
```
Comment (simonbray) [inline LICENSE:3]: @bgruening you think this line is ok?

Comment (bgruening) [reply LICENSE:3]: No idea, is that needed? I mean my name and the authors?

Comment (simonbray) [reply LICENSE:3]: No idea either, sorry :/

I just copied from IUC. It seems it is in https://mit-license.org/ and BioConda has that line as well.

Comment (bgruening) [reply LICENSE:3]: Then lets write:

> Copyright (c) 2012-present The 'galaxytoools' contributors

Comment (bgruening) [reply LICENSE:3]: Thanks

**Review by bgruening — APPROVED** [review]

**Pull request merged**